applications/salk-portal/src/components/common/BaseDialog.tsx

```diff
@@ -34,7 +34,7 @@ const Modal: React.FC<IModal> = ({
             Cancel
           </Button>
           <Button
-            disableElevation="true"
+            disableElevation
```

Comment:
please run `npm run build` then you will see that this is not according the code quality standards. An attribure should have a value